### ISSUE TITLE
[SOLVED] [BUG] How to update FVM package itself? Command didn't work. (Not Flutter version)

### ISSUE THREAD

**Issue by yasinarik** [leoafarias/fvm#283]
I've been using FVM for a long time, no issues, great plugin! Thanks a lot :)

I needed to upgrade my Flutter version. Then I have seen that FVM has many updates. So I wanted to upgrade the FVM plugin itself. I read the docs carefully but couldn't figure how to update out.

Anyways, I have installed a newer version of Flutter `2.2.0-10.2.pre` using FVM. It was installed successfully and showed an output message as below:

```
FVM Update Available 1.3.6 → 2.0.1 
Changelog: https://pub.dev/packages/fvm/changelog
Run pub global activate fvm to update
```

So I run `pub global activate fvm` . It runs but the current version **doesn't change.** 

What should I do? It sounds more like a question but I don't know, is the docs lack something about updating or I did something wrong?

**Comment by kuhnroyal** [comment]
You need to update the global Dart installation, the one you are running pub from. It is probably pre Dart 2.10

**Comment by yasinarik** [comment]
**OK,** let me quickly tell you what I did. I needed to migrate to `null safety`

- Installed `Flutter 2.2.0-10.2.pre` version by using `fvm install`. It was `1.25.0-4.0.pre` before. Then I run `fvm use` and picked the newer version.
- Changed the versions of all of the packages to be compatible with null safety. Thankfully, they all have a version to support at the moment.
- Changed the .yaml like this: 

**It was**
```yaml
environment:
  sdk: ">=2.7.0 <3.0.0"
```

**I've changed it to**
```yaml
environment:
  sdk: ">=2.12.0 <3.0.0"
```

- Result of `dart --version`:
```
Dart SDK version: 2.10.4 (stable) (Wed Nov 11 13:35:58 2020 +0100) on "macos_x64"
```

- Result of `dart pub upgrade` and `dart pub get` (both of the same):
```
Resolving dependencies... 
The current Dart SDK version is 2.10.4.

Because myprojectnamehere requires SDK version >=2.12.0 <3.0.0, version solving failed.
```

Currently,  I am stuck. Can't upgrade. I of course ran `fvm flutter clean`, `fvm flutter pub get` etc.
